### DISCUSSION TITLE
Text Field Input Type - Number

### DISCUSSION THREAD

**Discussion by jakobwinter** [vuetifyjs/vuetify#13237] [Q&A]
_I'm quite new to Vue and Vuetify, so I might just be missing the obvious..._ 

When using the Text Field Component, one gets a really nicely formatted input field. And when you set the **type="time"** it gets even more awesome, with nice formatting of the input value and a little X-button to clear the input.

When setting type="number" however, it looks quite horrible. An up- and a down-arrow are added, but they are standard html formatting, no nice vuetify styling. Also, whereas the Text Field with type time only accepts numbers, the Text Field with type number, also accepts letters.

So my question is: **What different types are there** and am I simply missing the correct one for a beautifully layouted number input?

And if there isn't a number-input type. Why not??? Surely I'm not the only one in need of this functionality...

_Really hoping I am simply missing the obvious here..._

Cheers,
Jakob

**Comment by nekosaur** [comment]
Whatever you get when using the `type` property is browser dependent. Vuetify does not do anything special outside of that. The available types can be found here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

**Reply by jakobwinter** [reply]
Ah, I see. Thanks for the swift reply. 
But wouldn't that be a useful feature, to have nicely formatted numbers-only input natively in Vuetify?

What would you do to tackle this problem?

**Reply by nekosaur** [reply]
There is a feature request here https://github.com/vuetifyjs/vuetify/issues/7374

**Reply by jakobwinter** [reply]
Awesome. Thanks!
